tools/fedora/cp2k.spec

```
@@ -250,7 +247,7 @@ for mpi in '' mpich %{?with_openmpi:openmpi} ; do
   # Run packaged do_regtest.sh with appropriate buildroot runpaths
   env PATH=%{buildroot}${bindir}:${PATH} \
     LD_LIBRARY_PATH=%{buildroot}${libdir} \
-    tests/do_regtest.py ${test_common_args[@]} ${test_mpi_args[@]}
+    tests/do_regtest.py %{_bindir} ${test_mpi_args[@]}
```

```suggestion
    tests/do_regtest.py %{buildroot}${bindir} ${test_mpi_args[@]}
```
If you want to point it to "installed" path
```suggestion
    tests/do_regtest.py %{_vpath_builddir}/bin ${test_mpi_args[@]}
```
If you want to point it to the build path